protocol/messenger_pin_messages.go

```diff
@@ -73,24 +73,6 @@ func (m *Messenger) sendPinMessage(ctx context.Context, message *common.PinMessa
 }
 
 func (m *Messenger) PinnedMessageByChatID(chatID, cursor string, limit int) ([]*common.PinnedMessage, string, error) {
-	chat, err := m.persistence.Chat(chatID)
```

Comment:
Is this intentional?

Reply:
yes, we don't have pinned messages for timeline